ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -362,6 +362,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-upi
+- as: vsphere-ipi-disconnected-p2-f7
+  cron: 38 13 30,7,14,21 * *
+  steps:
+    cluster_profile: vsphere
```

Comment:
need to use cluster_porfile `vsphere-discon`

Reply:
fixed